examples/sections/index.html

```diff
@@ -8,7 +8,7 @@
     <link rel="stylesheet" href="css/ul.css">
     <link rel="stylesheet" href="css/app.css">
 
-    <script src="bower_components/dom-scheduler/lib/dom-scheduler.js" defer></script>
+    <!-- <script src="bower_components/dom-scheduler/lib/dom-scheduler.js" defer></script> -->
```

Comment:
not sure, maybe the section with the scheduler and the simple one without?
(both with an easy comment to toggle on/off)